spec/javascripts/NewService/ServiceDiscoveryForm.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Enzyme, {shallow} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import ServiceDiscoveryForm from 'NewService/components/ServiceDiscoveryForm.jsx'
+
+Enzyme.configure({adapter: new Adapter()})
+
+describe('Service Discovery Form', () => {
+  const Props = {
+    isVisible: true,
+    providerAdminServiceDiscoveryServicesPath: 'my-path',
+    namespaces: ['one', 'two', 'three'],
+    services: ['a', 'b', 'c'],
+    selectedNamespace: 'one',
+    handleChangeNamespaces: jest.fn()
+  }
+
+  it('should render properly ', () => {
```

Comment:
it should render properly, I'm expecting some specifics not only the shallow component exists